todolist/settings.py

```diff
@@ -62,12 +62,12 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
     'default': {
-        'ENGINE': 'mysql.connector.django',
-        'NAME': 'app_db',
-        'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': 'mysql',  # You can use a different host if your MySQL server is on a remote machine.
-        'PORT': '',  # Leave this empty to use the default MySQL port (3306).
+        'ENGINE': os.getenv('ENGINE', 'django.db.backends.sqlite3'),
+        'NAME': os.getenv('NAME', os.path.join(BASE_DIR, 'db.sqlite3')),
+        'USER': os.getenv('USER', ''),
+        'PASSWORD': os.getenv('PASSWORD', ''),
+        'HOST': os.getenv('HOST', ''),
+        'PORT': os.getenv('PORT', ''),
```

Comment:
The database configuration uses environment variables, which is a good practice. Ensure these environment variables (`ENGINE`, `NAME`, `USER`, `PASSWORD`, `HOST`, `PORT`) are correctly set in your deployment environment.